plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2530,6 +2561,12 @@ public void testIncorrectIcebergFileSizes()
         dropTable("test_iceberg_file_size");
     }
 
+    protected BufferedInputStream createDataFileStreamForFileSystem(String filePath)
+            throws FileNotFoundException
+    {
+        return new BufferedInputStream(new FileInputStream(filePath));
```

Comment:
Does it matter for the caller that it's BufferedInputStream?
let's have the method return InputStream. if caller needs, it can wrap